src/rust/ide/src/view/layout.rs

```diff
@@ -96,25 +98,33 @@ impl ViewLayout {
     , visualization_controller : controller::Visualization
     , fonts                    : &mut font::Registry
     ) -> FallibleResult<Self> {
-        let logger           = logger.sub("ViewLayout");
-        let world            = &application.display;
-        let text_editor      = TextEditor::new(&logger,world,text_controller,kb_actions,fonts);
-        let graph            = graph_controller.graph.clone_ref();
-        let node_searcher    = NodeSearcher::new(world,&logger,graph,fonts);
-        let graph_controller = graph_controller.clone_ref();
-        let node_editor      = NodeEditor::new
+        let logger        = logger.sub("ViewLayout");
+        let world         = &application.display;
+        let text_editor   = TextEditor::new(&logger,world,text_controller,kb_actions,fonts);
+        let graph         = graph_controller.graph.clone_ref();
+        let node_editor   = NodeEditor::new
             (&logger,application,graph_controller,visualization_controller).await?;
+        let node_searcher = NodeSearcher::new(world,&logger,node_editor.clone_ref(),graph,fonts);
         world.add_child(&text_editor.display_object());
         world.add_child(&node_editor);
         world.add_child(&node_searcher);
-        let size = zero();
-        let data = ViewLayoutData {text_editor,node_editor,node_searcher,size,logger};
+        let size           = zero();
+        let mouse_position = world.scene().mouse.position.clone_ref();
+        let data = ViewLayoutData{text_editor,node_editor,node_searcher,size,logger,mouse_position};
         let rc   = Rc::new(RefCell::new(data));
         Ok(Self {rc}.init(world,kb_actions))
     }
 
-    fn init_keyboard(self, _keyboard_actions:&mut keyboard::Actions) -> Self {
+    fn init_keyboard(self, keyboard_actions:&mut keyboard::Actions) -> Self {
         // TODO[ao] add here some useful staff (quitting project for example)
+        let layout = self.rc.clone();
+        keyboard_actions.add_action(&[keyboard::Key::Tab], move || {
+            let mut layout             = layout.borrow_mut();
+            let position               = layout.mouse_position.get();
+            let node_searcher_position = Vector3::new(position.x as f32,position.y as f32,0.0);
```

Comment:
This won't work after panning the scene. As this is temporary code this is acceptable, but there should be left a comment with an explanation.

Reply:
Same will go for the generated node's initial position.

Reply:
@mwu-tow I don't think it's a valid concern because the node searcher will also move with panning, but I will leave a note for testing.